src/app/modules/dashboard/vault-stats/vault-stats.component.ts

```diff
@@ -0,0 +1,330 @@
+import {ChangeDetectorRef, Component, OnInit} from '@angular/core';
+import { ChartData } from './mokData/charts.data';
+import * as echarts from 'echarts';
+
+@Component({
+  selector: 'app-vault-stats',
+  templateUrl: './vault-stats.component.html',
+  styleUrls: ['./vault-stats.component.css']
+})
+export class VaultStatsComponent implements OnInit {
+
+    options1: any;
+    options2: any;
+
+    private data1: any[] = [];
+    private data2: any[] = [];
+    title1 = 'TVL';
+    title2 = 'Profits to Farmers';
+    tvlSelectedValue = '';
+    tvlSelectedDate = '';
+    profitSelectedValue = '';
+    profitSelectedDate = '';
+    changesVolumInPercent = 0;
+    changesVolumInAmount = 0;
+    changesTvlInPercent = 0;
+    changesTvlInAmount = 0;
+    minus1 = false;
+    minus2 = false;
+
+
+    constructor(private ref: ChangeDetectorRef) { }
+
+    ngOnInit(): void {
+        this.setTVLDefault();
+        this.setProfitDefault();
+        this.fillchartsDataSets();
+        this.initializeCharts();
+        this.lastChanges();
+    }
+
+    lastChanges(): void {
+        const lastChangesVolume = +ChartData[ChartData.length - 1].volumeUSD - +ChartData[ChartData.length - 2].volumeUSD;
```

Comment:
please, don't use ChartData inside this class
it should be reusable and ready to work with any data
you can create another component with an example implementation